blog/docker-commit-convert-a-container-into-an-image.md

```diff
@@ -0,0 +1,30 @@
+# Docker commit: convert a container into an image
+
+We can convert a specific container into a base Docker image. It's not the recommended way to create images because these could include useless cached or session data. The easier way to control the data that it's stored into the Docker layers is using the `Dockerfile`.
+
+🧠 Remember that a Docker image has a read-only filesystem based on layers and works as a _template_ to start containers in a preinstalled environment, and a Docker container uses this _"template"_ to run isolated applications into a virtualized run-time environment with a normal filesystem.
+
+## Do we need to use this to create images with non-root users?
+
+No, you can run commands with another users using the `Dockerfile` as well, you can see an example [here](https://dev.to/cloudx/testing-our-package-build-in-the-docker-world-34p0).
+
+## Adding a port forwarding rule to a container
+
+Perhaps we are testing a random software with a container and we need to publish a port to the host OS, or to another computer.
```

Comment:
```suggestion
It can occur that we are testing a random software within a container and we need to publish a port to the host OS, or to another computer.
```